docs/source/deployment/corda-health-checker.rst

```diff
@@ -0,0 +1,162 @@
+
+Report Format
+-------------
+
+After each run, the Corda Health Survey collects and packages up into a .zip file information that R3 Support can use to help a customer with a support request, including:
+
+- An obfuscated version of the config files (i.e., without passwords, etc.)
+- Node logs from the last 3 days (if the user is happy to share)
+- The version of Corda, Java virtual machine and operating system, networking information with DNS lookups to various endpoints (database, network map, doorman, external addresses)
+- A copy of the network parameters file
+- A list of installed CorDapps (including file sizes and checksums)
+- A list of the files in the drivers directory
+- A copy of the Node information file and a list of the ones in the additional-node-infos directory, etc.
+- Instead of zipping the reports, operators can print them to a text file using the command line option -t.
+
+The tool optionally allows operators to upload the resulting report to a support ticket. JIRA credentials can either be provided through the environment variables JIRA_USER and JIRA_PASSWORD, or through the prompt.
+
+Disabling the Corda Health Survey in Production
+-----------------------------------------------
+
+The tool relies on dedicated Artemis queues to relay configuration and runtime information from the Corda Firewall components. This functionality is enabled by default. After verifying a production deployment, operators are advised to disable the health checking functionality (in order to use the standard Artemis setup for Corda Enterprise) by adding the following entry in the Node configuration file:
+
+.. sourcecode:: shell
+
+	enterpriseConfiguration { healthCheck = false }
+
+And the following entry in the Bridge configuration file:
+
+.. sourcecode:: shell
+
+	healthCheck = false
+
+Running the Corda Health Survey
+-------------------------------
+
+The tool runs with the node installation as its current working directory, e.g.:
+
+.. sourcecode:: shell
+
+	$ cd /opt/corda/
+	$ java -jar corda-tools-health-survey.jar
+
+Alternatively, the base directory and Node/Bridge/Float configuration paths can be specified as command-line arguments:
+
+base-directory or -d, specifying the path to the Node installation.
+node-configuration or -c, specifying the path to the Node configuration file.
+
+These options can be specified together, allowing for setups with Node configuration files under separate directories.
+
+The majority of the checks require the Node (and related Firewall components, if present) to have been started before the tool is run. Please refer to the Corda Enterprise documentation on how to do this.
+
+Other available command-line arguments include:
+
+.. sourcecode:: shell
+
+	--local or -l, verify local Node configuration only without checking Bridge/Float, by default verifies all.
```

Comment:
This will eventually end up in docs.corda.net as reference documentation and should be removed from this site when that happens.